backend/api/Controllers/MissionController.cs

```diff
@@ -96,19 +152,38 @@ IStidService stidService
     }
 
     /// <summary>
-    /// Lookup mission by specified id.
+    /// Lookup mission run by specified id.
     /// </summary>
     [HttpGet]
     [Authorize(Roles = Role.Any)]
-    [Route("{id}")]
-    [ProducesResponseType(typeof(Mission), StatusCodes.Status200OK)]
+    [Route("runs/{id}")]
+    [ProducesResponseType(typeof(MissionRun), StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status403Forbidden)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-    public async Task<ActionResult<Mission>> GetMissionById([FromRoute] string id)
+    public async Task<ActionResult<MissionRun>> GetMissionRunById([FromRoute] string id)
     {
-        var mission = await _missionService.ReadById(id);
+        var mission = await _missionRunService.ReadById(id);
+        if (mission == null)
+            return NotFound($"Could not find mission with id {id}");
+        return Ok(mission);
+    }
+
+    /// <summary>
+    /// Lookup mission definition by specified id.
+    /// </summary>
+    [HttpGet]
+    [Authorize(Roles = Role.Any)]
+    [Route("definitions/{id}")]
+    [ProducesResponseType(typeof(MissionRun), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<MissionDefinition>> GetMissionDefinitionById([FromRoute] string id)
+    {
+        var mission = await _missionDefinitionService.ReadById(id);
         if (mission == null)
             return NotFound($"Could not find mission with id {id}");
```

Comment:
Should be missionDefinition:
`return NotFound($"Could not find missionDefinition with id {id}");`